easybuild/easyconfigs/m/MotionCor2/MotionCor2-1.4.4-GCCcore-10.3.0-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,54 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Ake Sandgren
+# HPC2N
+# Umea University
+
+name = 'MotionCor2'
+version = '1.4.4'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://msg.ucsf.edu/'
+description = """MotionCor2 correct anisotropic image motion at the
+single pixel level across the whole frame, suitable for both single
+particle and tomographic images. Iterative, patch-based motion detection
+is combined with spatial and temporal constraints and dose weighting.
+
+Cite publication: Shawn Q. Zheng, Eugene Palovcak, Jean-Paul Armache,
+Yifan Cheng and David A. Agard (2016) Anisotropic Correction of
+Beam-induced Motion for Improved Single-particle Electron
+Cryo-microscopy, Nature Methods, submitted.
+BioArxiv: https://biorxiv.org/content/early/2016/07/04/061960
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+# No longer directly downloadable, available from https://msg.ucsf.edu/software
+sources = [
+    '%(name)s_%(version)s.zip',
+]
+checksums = ['128f5cdf83b5023204c10b4801baa13f0a56a950e0bc6e0ed26c7f3aa1df59d7']
+
+# CUDA is a build dependency to make sure it gets installed.
+# It's actually a runtime dependency, but that's handled in the wrapper to
+# make sure it doesn't conflict with whatever toolchain happens to be loaded.
+# Change CUDA version to match the latest one used in this version
+# of MotionCor2
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('patchelf', '0.12'),
+    ('CUDA', '11.3.1', '', True),
+]
+
+skip_rpath_check = True
+
+postinstallcmds = ['patchelf --force-rpath --set-rpath $EBROOTCUDACORE/lib64:$EBROOTLIBTIFF/lib %(installdir)s/bin/MotionCor2_1.4.4_Cuda113']
```

Comment:
This should probably only be done if the rpath easyconfig config option is set to true